crossdata-common/src/main/scala/com/stratio/crossdata/common/Message.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2015 Stratio (http://stratio.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.stratio.crossdata.common
+
+case class Message(query: String)
+case class ExecuteQuery(query:String)
```

Comment:
ExecuteQuery seems to be an internal server message. I'd move it to ExecutorActor.